src/scripts/Selector/SelectorTable.js

```diff
@@ -49,7 +49,7 @@ export default class SelectorTable extends Selector {
 	getVerticalDataCells(table) {
 		const columnIndices = this.getTableHeaderColumns($(table));
 		const dataSelector = this.getTableDataRowSelector();
-		const dataColumns = this.getDataColumns();
+		const dataColumns = this.getDataColumns(); // Столбцы имеющие флаг extract == True
```

Comment:
- Please write comments in English
- If your comment is relevant for method rather than its invocation, move it to method definition
- Don't write redundant comments. You don't have to clarify that `columns.filter(column => column.extract)` selects columns with `extract == True`, this is exactly what is written in the code. Prefer writing self-explanatory code to adding comments.